assets/js/admin/toggle-edit-organization.js

```diff
@@ -0,0 +1,8 @@
+const editOrganizationBtn = document.querySelector('.js-edit-organization__btn');
```

Comment:
This seems project specific, can it be removed?

Reply:
removed https://github.com/netgen/media-site/commit/c4c897f58797d6772823557b30c8fcde98037e10